doc/qbk/3.0.parsing.qbk

```diff
@@ -7,6 +7,17 @@
     Official repository: https://github.com/CPPAlliance/url
 ]
 
+[section URL]
+
+[h5 Notation]
+
+This library standardizes on the term "URL" to refer to all strings which
+follow the syntax in __rfc3986__. The reason is that the terms URI and IRI
+cause confusion. In practice a single algorithm is used for both so
+keeping them distinct yields little value. When discussing particular
```

Comment:
"yields little value" is subjective

Reply:
You could say "as the terms URI and IRI and not widely recognized, this library standardizes on the term 'URL' to refer to any string which meets any of the syntactical requirements."

Reply:
Something that looks a little weird from my ignorant perspective is that "this library standardizes on the term 'URL'" but the very next paragraph lists the functions for parsing URLs and none of them use the term `url`. I understand it does that because of the grammar, but it's still weird in this context. 

It's like it deviates from the grammar to standardize on "URL", then all functions use the term `uri` not to deviate from the grammar, and then return `url` or `url_view`, deviating from the grammar again. Especially because it doesn't return a `uri_view` or something, to not deviate from the grammar again. 

Maybe it would be better to say nothing about standardizing on URL.

Reply:
> Maybe it would be better to say nothing about standardizing on URL.

Yes there is merit to this option